src/components/TabletApp/RightSide/QuickBook/QuickBook.tsx

```diff
@@ -4,53 +4,94 @@ import {
     Button,
     Typography,
     Grid,
-    Paper,
     TextField,
     Autocomplete,
 } from "@mui/material";
 import EditCalendarIcon from "@mui/icons-material/EditCalendar";
-import { useState, useEffect } from "react";
+import { useState } from "react";
 import axios from "axios";
-import { styled } from "@mui/material/styles";
-import { getParticipants } from "../../../../api/getRequests";
-import COLORS from "../../../../constants/CustomColors";
-
-//To be removed when globla theme is done
-const Item = styled(Paper)(({ theme }) => ({
-    backgroundColor: theme.palette.mode === "dark" ? "#1A2027" : "#fff",
-    ...theme.typography.body2,
-    padding: theme.spacing(1),
-    textAlign: "center",
-    color: theme.palette.text.secondary,
-}));
-
-//To be removed when global theme is done
-const buttonStyle = { color: COLORS.GREEN, border: `1px solid ${COLORS.GREEN}` };
+import { getParticipants, getParticipant } from "../../../../api/getRequests";
+import dayjs from "dayjs";
+import { spawnToast } from "../../../../utils/Toast";
+
+const Owner = {
+    name: "",
+    id: 0,
+    available: true,
+    image: "",
+};
```

Comment:
- constants name are usually defined with capital letters (OWNER)
- constants name should be suggestive, so here you define a default / initial owner so change the name to something more suggestive

Reply:
changed Owner to INITIALOWNER